skimage/segmentation/active_contour_model.py

```diff
@@ -48,6 +54,16 @@ def active_contour(image, snake, alpha=0.01, beta=0.1,
         Maximum iterations to optimize snake shape.
     convergence: float, optional
         Convergence criteria.
+    callback: function, optional
+        Called every 11 iterations with three args.

```

Comment:
11 is a strange number. (Why not 10?) In fact, it is called every time convergence is tested as well. Should we say this instead?